infrastructure/aws/mkscratch.sh

```diff
@@ -20,7 +20,7 @@ set -o pipefail # Check all commands in a pipeline
 #       "ProductName" : "Unknown Device",
 #       "SerialNumber" : "AWS143416FC5A55CA413",
 #       "UsedBytes" : 300000000000,
-#       "MaximiumLBA" : 585937500,
+#       "MaximumLBA" : 585937500,
```

Comment:
These were actually verbatim output from nvme-cli, but that apparently was fixed a while ago in https://github.com/linux-nvme/nvme-cli/pull/397 and presumably we'll see the changes eventually, so there's no harm in correcting these.  Thanks!